packages/admin/src/services/ApplicationService.ts

```diff
@@ -71,13 +72,30 @@ const getApplicationFormSummary = async (
 const getApplicationFormSection = async (
   applicationId: string,
   sectionId: string,
-  sessionId: string
+  sessionId: string,
+  isV2Scheme?: boolean
 ) => {
-  const response = await axios.get(
-    `${BASE_APPLICATION_URL}/${applicationId}/sections/${sectionId}`,
+  const sectionsNeedMapped =
+    isV2Scheme &&
+    ['ORGANISATION_DETAILS', 'FUNDING_DETAILS'].includes(sectionId);
+
+  const { data } = await axios.get<ApplicationFormSection>(
+    `${BASE_APPLICATION_URL}/${applicationId}/sections/${
+      sectionsNeedMapped ? 'ESSENTIAL' : sectionId
+    }`,
     axiosSessionConfig(sessionId)
   );
-  return response.data as ApplicationFormSection;
+
+  let v2SectionData = null;
```

Comment:
Any reason why you don't do this? 

` if (!sectionsNeedMapped) {
    return null;
  }

  return mapSingleSection(
      data,
      sectionId as 'ORGANISATION_DETAILS' | 'FUNDING_DETAILS'
    );`

Reply:
Nice thanks, have pushed that